app/components/provider_interface/course_summary_component.rb

```diff
@@ -30,6 +30,17 @@ def rows
           value: study_mode,
         },
       ]
+      return rows unless course_option.course&.accredited_provider&.name_and_code
```

Comment:
Nit, `.accredited_provider` may return `nil` but is always available

```suggestion
      return rows unless course_option.course.accredited_provider.present?
```